hal/lib/athena/handles/HandlesInternal.h

```diff
@@ -23,6 +23,8 @@
 
 namespace hal {
 
+constexpr int16_t InvalidGetHandle = -1;
```

Comment:
Maybe.  You're right, it's not a handle value at all.  It's an invalid index?  Maybe `InvalidHandleIndex`?  I prefer naming constants after the type they're placeholding, not after the function..